test/asserts/routing-number-assert_test.js

```diff
@@ -0,0 +1,41 @@
+
+/**
+ * Module dependencies.
+ */
+
+import { Assert as BaseAssert, Violation } from 'validator.js';
+import RoutingNumberAssert from '../../src/asserts/routing-number-assert';
+import should from 'should';
+
+/**
+ * Extend `Assert` with `RoutingNumberAssert`.
+ */
+
+const Assert = BaseAssert.extend({
+  RoutingNumber: RoutingNumberAssert
+});
+
+/**
+ * Test `RoutingNumberAssert`.
+ */
+
+describe('RoutingNumberAssert', () => {
+  it('should throw an error if `value` is invalid', () => {
```

Comment:
This should be separated into multiple tests, such as input value type test, invalid input type and assert name.